tests/boardfarm_plugins/boardfarm_prplmesh/tests/initial_ap_config.py

```diff
@@ -4,13 +4,31 @@
 # See LICENSE file for more details.
 
 from boardfarm.tests import bft_base_test
+from utils import Utils
 
 
 class InitialApConfig(bft_base_test.BftBaseTest):
-    """PrplMesh sample test case, no actual testing, it just passes."""
+    """Check initial configuration on device."""
 
     def runTest(self):
-        """Main test logic must be implemented here."""
+        for dev in self.dev:
+            if dev.agent_entity:
+                dev.wired_sniffer.start(self.__class__.__name__)
```

Comment:
This should be refactored to be done automatically before each test. But that can be done later.